.eslintignore

```diff
@@ -5,7 +5,7 @@ coverage/
 
 node_modules/
 
-src/assets/
+packages/velaux-ui/src/assets/
```

Comment:
let's also add `.DS_Store` file into gitignore

Reply:
It exists. This is eslint file.